### ISSUE TITLE
Sceneitem_factory wrong return

### ISSUE THREAD

**Issue by Dschogo** [Julian-O/PySLOBS#7]
Hey, i've noticed that the transform_postition of an Item is wrong.

instead of excpected something like:
IVec2(x=50, y=0)  (with two integers, one as x and one as y)

it was:

IVec2(x='x', y='y') 

Strings instead of the actual parameters in the tuple.

The problem is in the sceneitem_factory (factories.py line 33)

The faulty part is the else:
instead of this:

```py
def sceneitem_factory(connection, json_dict):
    # We see inconsistency in choices of API here.
    if isinstance(json_dict["transform"]["position"], list):
        transform_position = (
            IVec2(
                x=json_dict["transform"]["position"][0],
                y=json_dict["transform"]["position"][1],
            ),
        )
    else:
        transform_position = IVec2(*json_dict["transform"]["position"])
```

It should be that:

```py
def sceneitem_factory(connection, json_dict):
    # We see inconsistency in choices of API here.
    if isinstance(json_dict["transform"]["position"], list):
        transform_position = (
            IVec2(
                x=json_dict["transform"]["position"][0],
                y=json_dict["transform"]["position"][1],
            ),
        )
    else:
        transform_position = (
            IVec2(
                x=json_dict["transform"]["position"]['x'] ,
                y=json_dict["transform"]["position"]['y'] ,
            )
        )
```

Should i create a PR for that?

**Comment by Julian-O** [comment]
Thanks for catching and reporting this!

At first blush, it appears that I meant to have two asterisks, rather than one. - i.e. `IVec2(**json_dict[...])`

I hope to get a chance in the next couple of days to write some tests to reproduce, fix the bug, look to see if I have made the same mistake elsewhere, maybe push the coverage needle a bit further to the right, and make a new release.

I really appreciate the detailed bug report. Thanks again.

**Comment by Julian-O** [comment]
Fixed in 1.2.2.
